test/graphcut/normalized_cut.jl

```diff
@@ -0,0 +1,24 @@
+@testset "Normalized Cut" begin
+
+    g = SimpleWeightedGraph(4)
+    add_edge!(g, 1, 2, 1)
+    add_edge!(g, 3, 4, 1)
+
+    labels = ncut(g.weights, 0.1)
```

Comment:
See the other test scripts: basically we loop through graphs of different eltypes:
```
gx = SimpleWeightedGraph(4)
add_edge!(gx, 1, 2, 1)
add_edge!(gx, 3, 4, 1)

for g in testgraphs(gx)
   labels = ncut(g.weights, 0.1)
   @test ....
end
```